cpp/include/cudf/detail/get_value.cuh

```diff
@@ -26,7 +26,8 @@ namespace detail {
 /**
  * @brief Return a fixed-width value from column.
  *
- * This will retrieve the specified value from device memory.
+ * This will retrieve the specified value from device memory. This function
```

Comment:
One of my pet peeves is using future tense in documentation. Let's switch this to present tense, it is nearly always clearer. Plus, with your addition the tenses are mixed.

I know, I know, this is very nit-picky.

```suggestion
 * Retrieves the specified value from device memory. This function
```